tutorials/proto_rename/renaming-proto.md

```diff
@@ -25,42 +25,42 @@ Contains main method and methods to write and read a proto file in binary format
 ```bash
 bazel build //tutorials/proto_rename:all
 ```
-- Look at `writePerson()` method of <walkthrough-editor-open-file 
+- Look at `writePersons()` method of <walkthrough-editor-open-file 
                                     	    filePath="startup-os/tutorials/proto_rename/PersonTool.java" 
                                     	    text="PersonTool.java">
                                         </walkthrough-editor-open-file> class. 
-This method is creating a new Person which will be saved to `person.protobin` file.
-- Run the command to create `person.protobin` file:
+This method is creating a new Persons which will be saved to `persons.protobin` file.
+- Run the command to create `persons.protobin` file:
 ```bash
 bazel run //tutorials/proto_rename:person_tool -- write
 ```
-- Run the command to read `person.protobin` file:
+- Run the command to read `persons.protobin` file:
 ```bash
 bazel run //tutorials/proto_rename:person_tool -- read
 ```
 
 ## Renaming scalar field
```

Comment:
Let's not mention it's a scalar field, as this example works the same for enum or message fields.

Reply:
Done.